docs/changehistory/NextVersion.md

```diff
@@ -5,22 +5,23 @@ publish: false
 
 Table of contents:
 
-- [Revert timeline changes](#revert-timeline-changes)
-- [Display](#display)
-  - [Instancing](#instancing)
-  - [Overriding line color](#overriding-line-color)
-  - [Context Reality model visibility](#context-reality-model-visibility)
-  - [Contour Display](#contour-display)
-- [Interactive Tools](#interactive-tools)
-  - [Element Locate](#element-locate)
-- [Presentation](#presentation)
-  - [Calculated properties specification enhancements](#calculated-properties-specification-enhancements)
-- [Quantity](#quantity)
-- [API Deprecations](#api-deprecations)
-  - [@itwin/appui-abstract](#itwinappui-abstract)
-  - [@itwin/core-backend](#itwincore-frontend)
-  - [@itwin/core-frontend](#itwincore-frontend)
-  - [@itwin/core-quantity](#itwincore-quantity)
+- [NextVersion](#nextversion)
```

Comment:
@franklbh can you delete this line, reverting the table of contents? When I tried deleting the line locally, it kept undoing the delete...